app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<div class="container_form">
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field form_input">
+    <%= f.email_field :email, placeholder: "Email", autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field form_input">
+    <%= f.password_field :password, placeholder: "Password", autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field form_input">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in", class: "button" %>
+  </div>
+<% end %>
```

Comment:
- [ ] Great effort so far! 👏 However, I noticed that the suggested change regarding redirecting a user to the home page after a successful sign-in hasn't been implemented yet. It's crucial for user experience that, upon signing in using their email and password, they should be seamlessly redirected to the home page of your application. 🏠 This not only enhances user navigation but also provides a smoother flow within your app. Please consider implementing this change to improve the overall user experience. 

Happy coding! 😊🚀🍀

Reply:
@Strangeal Hi there! Please, following the previous video, please find here one that shows that after the correct sign in is being redirect to the home path. 
https://www.loom.com/share/0bf1379b3adf481d9577d47ebf95f13d?sid=e5fea012-4278-4909-ad08-fdebaa2c3602